yesod-test/Yesod/Test.hs

```diff
@@ -693,6 +694,25 @@ get url = request $ do
     setMethod "GET"
     setUrl url
 
+-- | Follow a redirect, if the last response was a redirect.
+-- ==== __Examples__
+-- > get HomeR
+--
+-- > followRedirect
+followRedirect :: Yesod site
+               =>  YesodExample site ()
+followRedirect = do
+  mr <- getResponse
```

Comment:
I think it's a different case. When you don't put the CSRF token in, it's almost always a mistake, and the messages are a useful way to find that out. In the redirect case (at least in my use case), you may actually want to check that a redirect has not happened, so it's easier to inspect an Either type than it is to trawl through messages.